lib/ncr/approval_manager.rb

```diff
@@ -0,0 +1,90 @@
+module Ncr
+  class ApprovalManager
+    attr_reader :work_order
```

Comment:
I've been putting them all over this project! :) I don't have strong feelings per se but I do think it generally makes sense to keep the public API of an object as small as possible and making this private would do that.